src/reuse/modules/nonUi5/element.ts

```diff
@@ -61,7 +63,8 @@ export class ElementModule {
       vl.log(`wdio.waitForDisplayed invokation for selector ${selector}`);
       await $(selector).waitForDisplayed({ timeout: timeout });
     } catch (error) {
-      throw new Error(`Function 'waitToBeVisible' failed: ${error}`);
+      this.ErrorHandler.logException(error)
+      //throw new Error(`Function 'waitToBeVisible' failed: ${error}`);
```

Comment:
you can remove the old code from comments

Reply:
The changes reverted back and will add this changes in next PR